src/pages/workspaces/workspace/workflows/DataStepContent.js

```diff
@@ -44,16 +42,13 @@ export default class DataStepContent extends Component {
   }
 
   isValidSelectionModel() {
-    const { newSelectionModel: { newSetName, selectedEntities, type } } = this.state
-    const selectionSize = _.size(selectedEntities)
+    const { newSelectionModel: { newSetName, selectedEntities } } = this.state
+    return _.size(selectedEntities) > 0 && (!this.willCreateSet() || newSetName)
```

Comment:
Nit, but I found this logic easier to parse this way round:

```suggestion
    return _.size(selectedEntities) > 0 && (newSetName || !this.willCreateSet())
```

Reply:
Also - is this going to stop me if I choose to name my new set `"0"`? (perhaps the start of a sequence of sets...?)

Reply:
No, the string `'0'` is truthy. If we're dealing with numbers, that's when we need to worry about `0`, which is falsy.